### DISCUSSION TITLE
Welcome to cy-client Discussions!

### DISCUSSION THREAD

**Discussion by izqalan** [izqalan/cy-client#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bloodsucker3011** [comment]
how do you use the app ? is there any instruction ?

**Reply by izqalan** [reply]
You can download the program executable [here](https://github.com/izqalan/cy-client/releases/tag/v1.2.1) and run it like any other computer program and copy & paste album url in the box. 
If you want to download more than 1 album, press <kbd>Enter</kbd> to add another album url in a new line.

**Comment by bloodsucker3011** [comment]
Thank. I don't see the link.

On Fri, Sep 3, 2021 at 02:27 Izqalan N. ***@***.***> wrote:

> You can download the program executable here
> <https://github.com/izqalan/cy-client/releases/tag/v1.2.1> and copy &
> paste album url in the box.
> If you want to download more than 1 album, press Enter to add another
> album url in a new line.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/izqalan/cy-client/discussions/23#discussioncomment-1275400>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG5AW7ZJYBO5LCJDVR5MYGLUABTGHANCNFSM445T7ADQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
Thomas Ly

**Comment by Jvitormatos98** [comment]
How can i install this program on the Linux? i've download everything and did everything i could think of and still didn't work for me.
